hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -765,6 +765,14 @@ public class HoodieWriteConfig extends HoodieConfig {
       .withDocumentation("Avro schema of the partial updates. This is automatically set by the "
           + "Hudi write client and user is not expected to manually change the value.");
 
+  public static final ConfigProperty<Boolean> IGNORE_ERROR_WRITE = ConfigProperty
+          .key("hoodie.write.ignore.failed")
+          .defaultValue(true)
+          .markAdvanced()
+          .sinceVersion("1.0.0")
+          .withDocumentation("Whether ignore error and not throw exception when write occur errors."
```

Comment:
Whether to ignore the errors instead of throwing exception when the commit metadata encounters errors. if set as false job would error out, default is true

Reply:
done